docs/userguides/testing.md

```diff
@@ -127,7 +125,7 @@ def owner(accounts):
 
 @pytest.fixture
 def my_contract(project, owner):
-    # ^ use the 'project' fixture from the 'ape-test' plugin
+              # ^ use the 'project' fixture from the 'ape-test' plugin
```

Comment:
Is this indent intentional? I sense flake8 generally would not approve

Reply:
Good point, forgot about that! The intent was to line the `^` up with `project` 

Seems like it passed the linting but I can update it to have the indentation inside the comment rather than indent the comment itself. Or should we just leave the comment as-is you think?

Reply:
Since indents seems to always have to be lined up correctly in Python, my instinct is to keep it set next to the line below.

Also, I don't think the python linters will catch issues documentation examples.

I do see your point about it aligning with the project though. And since it's the docs, I can see the argument for keeping it like you put it.